src/dbnode/client/connection_pool.go

```diff
@@ -88,6 +90,7 @@ func newConnectionPool(host topology.Host, opts Options) connectionPool {
 	p := &connPool{
 		opts:               opts,
 		host:               host,
+		healthyGauge:       opts.InstrumentOptions().MetricsScope().Gauge("healthy"),
```

Comment:
Does this have the tag for the current host ID?  Otherwise we might not be able to discern it from other hosts?